src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import { Component } from 'react';
+import Calculator from './Calculator';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Kindly remove this` eslint-disable` linters line, it always a good practice to fix linters manually instead of disable it

Reply:
This change has been done well